scripts/bootstrap_manager.py

```diff
@@ -33,16 +33,20 @@ def run(config):
 
```

Comment:
If an exception is thrown somewhere, resources leak. There should be something like a big `try`, `except: clean up + raise/log`. This applies only to the `run` function.